services/adapter.ts

```diff
@@ -38,5 +38,8 @@ export const asyncWriteFile: (path: string, contents: string, options?: fs.Write
   fs.writeFile,
 );
 export const asyncChmod: (path: string, mode: fs.Mode) => Promise<void> = Bluebird.Promise.promisify(fs.chmod);
-export const asyncExists: (path: string) => Promise<boolean> = (path) => new Promise((resolve) => fs.exists(path, resolve));
+export const asyncExists: (path: string) => Promise<boolean> = (path) =>
+  new Promise((resolve) => {
+    fs.exists(path, resolve);
```

Comment:
`exists` is deprecated, we should use `stat`.

Reply:
Addressed in #175.